x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/dot_expander.tsx

```diff
@@ -26,6 +26,27 @@ const fieldsConfig: Record<string, FieldConfig> = {
         'Output field. Only required if the field to expand is part another object field.',
     }),
   },
+
+  /* Optional fields config */
```

Comment:
nit: isnt path (the above field) also optional? 🤔

Reply:
Yes, indeed it is, thanks for catching this! I will move the comment above the `path` field.